app/controllers/videos_controller.rb

```diff
@@ -15,4 +15,15 @@ def search
     @results = Video.search_by_title(params[:search_term])
   end
 
+  def create
+    @video = Video.new(video_params)
+    @video.save
```

Comment:
This action does not seem to be right, you already have it in admin and plus it does not make sense to have user to add new videos.